src/openforms/emails/digest.py

```diff
@@ -179,3 +190,36 @@ def collect_broken_configurations() -> list[BrokenConfiguration]:
         )
 
     return broken_configurations
+
+
+def collect_invalid_certificates() -> list[InvalidCertificate]:
+    today = timezone.now()
+
+    invalid_certs = []
+    for cert in Certificate.objects.all():
+        time_until_expiry = cert.expiry_date - today
+
+        if time_until_expiry <= timedelta(days=14):
+            invalid_certs.append(
+                InvalidCertificate(label=cert.label, expiry_date=cert.expiry_date)
+            )
+
+        if not (is_valid := cert.is_valid_key_pair()):
+            invalid_certs.append(
+                InvalidCertificate(label=cert.label, is_valid_pair=is_valid)
+            )
+
+    return invalid_certs
+
+
+def collect_invalid_configuration_overview() -> list[Entry]:
+    invalid_configuration_overview = ConfigurationCheck().get_configuration_results()
+
+    invalid_entries = [
+        entry
+        for item in invalid_configuration_overview
+        for entry in item["entries"]
+        if not entry.status
+    ]
+
+    return invalid_entries
```

Comment:
won't this report on plugins that aren't used at all too? those will definitely not pass config check, because why would you configure something you're not planning on using.

It'd probably be better to start from the forms and figure out:

* which prefill plugins are used
* which registration plugins are used (you already have this in another PR)
* which appointments module is used (this is configured globally)
* ...

and only test those that are actually used. But, we already have more specific detections for these things or they would be better anyway, so I'm not sure how useful it actually is to run the full configuration overview/check.

Reply:
Discussed on slack, this will not be added since we have more specific checks in place by now.